typings/index.d.ts

```diff
@@ -898,7 +898,12 @@ export interface Telegraf<TContext extends ContextMessageUpdate> extends Compose
    *
    * @param options [See reference to get more]{@link https://telegraf.js.org/#/?id=launch}
    */
-  launch(options?: Object): Promise<any>
+  launch(
+    options?: {
+      polling?: {timeout?: number, limit?: number, allowedUpdates?: tt.UpdateType[]}, 
```

Comment:
Can you add spaces, so that it looks like `{ noice: 'yeah' }` and not like `{noice: 'yeah'}`, please? Applies to the next line as well.

Reply:
Maybe options can be extracted to a separate interface as well? Just a thought.

Reply:
Included spaces.
Thought about including the interface but since the startPolling/startWebhook methods use positional arguments instead of object keys, they cannot use the interface. This restricts the helpfulness of the interface, and I think it's better to leave as-is unless the inputs to startPolling/startWebhook functions change to the same structure as launch()